config/crd/patches/webhook_in_clusterfilters.yaml

```diff
@@ -0,0 +1,16 @@
+# The following patch enables a conversion webhook for the CRD
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: clusterfilters.kubesphere.io
```

Comment:
Please check all crd/patches file if the domain is correct: `clusterfilters.kubesphere.io`? 
These files are auto-generated?